elyra/pipeline/kfp/kfp_processor.py

```diff
@@ -436,7 +422,7 @@ def process(self, pipeline):
 
             self.log_pipeline_info(
                 pipeline_name,
-                f"pipeline submitted: {public_api_endpoint}/#/runs/details/{run.id}",
+                f"pipeline submitted: {public_api_endpoint}/{experiment.experiment_id}/runs/{run.run_id}",
```

Comment:
This explicit change for ODH dashboard, not sure if it would work with kubeflow pipeline UI.
https://www.kubeflow.org/docs/components/pipelines/legacy-v1/overview/quickstart/#deploy-kubeflow-and-open-the-kubeflow-pipelines-ui